src/Azure.Functions.Cli/Helpers/ProjectHelpers.cs

```diff
@@ -40,16 +46,22 @@ public static string FindProjectFile(string path, LoggingFilterHelper loggingFil
                 {
                     foreach (FileInfo file in projectFiles)
                     {
-                        if (string.Equals(file.Name, Constants.ExtenstionsCsProjFile, StringComparison.OrdinalIgnoreCase)) continue;
-                        logger.LogDebug($"Found {file.FullName}. Using for user secrets file configuration.");
+                        if (string.Equals(file.Name, Constants.ExtensionsCsProjFile, StringComparison.OrdinalIgnoreCase)) continue;
+                        if (shouldLog)
```

Comment:
Did you mean to have this conditional logging change in this PR? Is that relevant to the NET7 publishing change?

Reply:
Yes, this change is to avoid unnecessary logging when attempting to find the function app's `.csproj` file, since we parse it to find the appropriate target framework when publishing. An alternative option I considered was making a copy of this method and removing the sections involving logging, but I believe this to be a more concise way to accomplish the same goal since:
1. The primary use of this method is to find the path to the `.csproj` file.
2. Any use of this method that does use logging should not be using `null` values for either `loggingFilterHelper` or `loggerFilterOptions`